filterAPI/setup.go

```diff
@@ -19,6 +21,14 @@ func setupInstance(instanceID string, update bson.M) error {
 	return nil
 }
 
+func setupDimensionOptions(ID string, update bson.M) error {
+	if err := mongo.Setup("datasets", "dimension.options", "_id", ID, update); err != nil {
```

Comment:
replace entire func with `return mongo.Setup("datasets", "dimension.options", "_id", ID, update)` would be a lot simpler?